lib/resources/content/tsconfig.json

```diff
@@ -10,7 +10,13 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "moduleResolution": "node",
-    "lib": ["es2017", "dom"]
+    "lib": ["es2017", "dom"],
+    "baseUrl": ".",
+    "paths": {
+      "*": [
+        "./src/*"
```

Comment:
This file is the one that actually is used to create the initial file on project creation, correct? If that's the case, then it would be fine that it is doing everything from `src` since users could overwrite it later if they have something different. It looks like there's a jsconfig file as well, should that be changed too? We want to make sure that behavior is going to be the same across implementations (TS, ES,...)